src/MetaModels/DcGeneral/Events/MetaModel/RenderItem.php

```diff
@@ -157,41 +167,6 @@ public static function getReadableValue(RenderReadablePropertyValueEvent $event)
     }
 
     /**
-     * Add additional parent header fields.
```

Comment:
Are you really sure this listener is not needed anymore? This is the parent header for parented MetaModels and not the child list.